baseplate/experiments/__init__.py

```diff
@@ -0,0 +1,103 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+import logging
+
+from .providers import experiment_from_config
+from .._compat import iteritems
+from ..context import ContextFactory
+from ..events import Event, EventTooLargeError, EventQueueFullError
+from ..file_watcher import FileWatcher, WatchedFileNotAvailableError
+
+
+logger = logging.getLogger(__name__)
+
+
+class ExperimentsContextFactory(ContextFactory):
+
+    def __init__(self, path, event_queue):
+        self._filewatcher = FileWatcher(path, json.load)
+        self._event_queue = event_queue
+
+    def make_object_for_context(self, name, server_span):
+        return Experiments(self._filewatcher, self._event_queue)
+
+
+class Experiments(object):
+
+    def __init__(self, config_watcher, event_queue=None, session=None):
+        self._config_watcher = config_watcher
+        self._event_queue = event_queue
+        self._cache = {}
+        self._session = session
+
+    def _get_config(self, name):
+        try:
+            config_data = self._config_watcher.get_data()
+            return config_data[name]
+        except WatchedFileNotAvailableError:
+            logger.warning("Experiment config file not found")
+            return
+        except KeyError:
+            logger.warning(
+                "Experiment <%r> not found in experiment config",
+                name,
+            )
+            return
+        except TypeError:
+            logger.warning("Could not load experiment config: %r", name)
+            return
+
+    def variant(self, name, **args):
```

Comment:
can this (and other similar ** usages) be named `kwargs` for consistency please?